### PR TITLE
[BDD] Behavior-Driven Development in Automated Testing

### PR DESCRIPTION
**Pull request by alex39292**

### CODE REVIEW

File: test/config/protractor_config.js [hunk 1]
```diff
@@ -7,20 +30,26 @@ exports.config = {
     framework: 'custom',
     frameworkPath: require.resolve('protractor-cucumber-framework'),
     capabilities: {
+        shardTestFiles: yargs.instances > 1,
+        maxInstances: yargs.instances,
         browserName: 'chrome',
         chromeOptions: {
-            args: ['--no-sandbox', '--window-size=1920,1080']
+            args: ['--no-sandbox', '--window-size=1920,1080'],
```
Comment (SiarheiRylach) [inline test/config/protractor_config.js:37]: nice!

File: test/features/home_page_title.feature [hunk 2]
```diff
@@ -0,0 +1,23 @@
+@smoke
+Feature: Raketa.by tests
+
+@cart
+Scenario: Test cart
+    Given I open "https://raketa.by/"
+    When I input "скричер" into searching field
+    When I click on the first item
```
Comment (SiarheiRylach) [inline test/features/home_page_title.feature]: Rule for When/Then. See in the our chat

File: test/features/home_page_title.feature [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    When I click Buy
+    When I go to the cart
+    When I check cost with real cost "84,90"
+
+@titles
+Scenario Outline: Page Titles <URL>
+    Given I open "<URL>"
+    Then Page title should be "<Title>"
+    When I wait "<Seconds>" seconds
```
Comment (SiarheiRylach) [inline test/features/home_page_title.feature]: Why wait in the end?

File: test/features/home_page_title.feature [hunk 4]
```diff
@@ -0,0 +1,23 @@
+@smoke
+Feature: Raketa.by tests
```
Comment (SiarheiRylach) [inline test/features/home_page_title.feature]: Description for the feature

File: test/features/home_page_title.feature [hunk 5]
```diff
@@ -0,0 +1,23 @@
+@smoke
```
Comment (SiarheiRylach) [inline test/features/home_page_title.feature:1]: html reports should be excluded from PR

File: test/config/protractor_config.js (hunk 1 above)
Comment (alex39292) [reply test/config/protractor_config.js:37]: :)

File: test/features/home_page_title.feature (hunk 3 above)
Comment (alex39292) [reply test/features/home_page_title.feature]: I don't know, it was in example. I can delete it, it's really unnecessary

File: test/features/home_page_title.feature (hunk 2 above)
Comment (alex39292) [reply test/features/home_page_title.feature]: Yeah, I see

File: test/features/home_page_title.feature (hunk 5 above)
Comment (alex39292) [reply test/features/home_page_title.feature:1]: excluded

File: test/features/home_page_title.feature (hunk 4 above)
Comment (alex39292) [reply test/features/home_page_title.feature]: ok

**Review by SiarheiRylach — APPROVED** [review]

**Pull request merged**